core/middlewares/wrap.js

```diff
@@ -117,7 +118,8 @@ exports.process = function (req, res, next) {
                     filename: templatePath
                 });
             } catch(err){
-                req.specData.renderedHtml = 'Error rendering Spec with EJS: ' + template;
+                console.log(err);
```

Comment:
I guess it was used for debugging? Please clean it, or add some prefix description like `console.log('wrap.js: error rendering spec with EJS', err);`